server/routes/fixtures/SuperTestWrapper.ts

```diff
@@ -0,0 +1,37 @@
+// eslint-disable-next-line import/no-extraneous-dependencies
+import request from 'supertest'
+// eslint-disable-next-line import/no-extraneous-dependencies
+import nock from 'nock'
+import express from 'express'
+import app from '../../index'
+import config from '../../config'
+
+jest.mock('redis', () => jest.requireActual('redis-mock'))
+
+export default class SuperTestWrapper {
+  request: request.SuperAgentTest
+
+  constructor() {
+    const application: express.Application = app()
+    this.request = request.agent(application)
+    this.request //
+      .set('Content-Type', 'application/x-www-form-urlencoded')
+      .redirects(1)
+  }
+
+  authenticateAsLegalSenderUser = async () => {
+    const mockedSendLegalMailApi = nock(config.apis.sendLegalMail.url)
+    mockedSendLegalMailApi.post('/link/verify', { secret: 'some-secret' }).reply(201, {
+      token:
+        'eyJhbGciOiJSUzI1NiJ9.eyJqdGkiOiJkZTE3NGM0Yi0xY2M2LTQxYWYtYTczYi01ZTE2YmI5YzE1ZWIiLCJzdWIiOiJtaWtlLmhhbG1hQGRpZ2l0YWwuanVzdGljZS5nb3YudWsiLCJleHAiOjQ3ODc1Njk3MjF9.WTqNajHRgZCbNe0g20lK5a7s_5-VeWD-FViu6gTgQaEsavimH_wEz1wZ4sj5osCDkCaLIgjYxGFt_p2IAsr7x0pI5b3CenN4_EMrz2pVVxAXOEEI8Q8QVfTy-iBGyO9W95rFGtmxbdsmYpr7LIr6DxJDUCCrCoeH8f4Dl-4QfKLUn-x_9_Bfum1rtAJ38B5pwiwhlzxeHD58C5XIc7swURGpCA97gtog7kEbyrCDF5AkIM4oYC1ViTMfDypnIJaDAU2ggxkaV5EkiIOB386POjUXkePQDnPajX3C-ugbJlKUPHp9z0CL_ngw5iK3wf9mEy2mWi9VHbUnyqVzfhrbIJK2PKQ0Fb8ZJIZhlB_rD68bgpaKskJwGy3lCMqDV5hiK5rUMsw_6n0asdYIhOvrEkXHrwmR4eRfobkLmtXGGRBswWuMhVXbYxBfZPU4PSkReTnbGRxSub-_UmMIvI_CXXaMdyRv0ixG4R3R7HfgLyZiTffN0p8nKmzKDXWWmPVJ',
+    })
+
+    await this.request.get('/link/verify-link?secret=some-secret')
+    nock.cleanAll()
+  }
+
+  unauthenticated = async () => {
+    await this.request.get('/link/request-link?force=true') // log out legal sender user
```

Comment:
The easiest way to be unauthenticated is to hit the 2 sign out links - it doesnt matter whether you were previously logged in as a LS or MR user, or neither.